### PR TITLE
MODBULKOPS-210: wrong Administrative note column header representation. Administrative note - value

### PR DESCRIPTION
**Pull request by DmytroBykov1**
## Purpose
https://issues.folio.org/browse/MODBULKOPS-210
Fix the wrong Administrative note column header representation (Bulk edit for Inventory - holdings).
Administrative note - the proper value
 

## Approach
Fix the preview header value for Administrative Note
 

## Pre-Merge Checklist:
Before merging this PR, please go through the following list and take appropriate actions.

- Does this PR meet or exceed the expected quality standards?
  - [x] Code coverage on new code is 80% or greater
  - [x] Duplications on new code is 3% or less
  - [x] There are no major code smells or security issues
- Does this introduce breaking changes?
  - [ ] Were any API paths or methods changed, added or removed?
  - [ ] Were there any schema changes?
  - [ ] Did any of the interface versions change?
  - [ ] Were permissions changed, added, or removed?
  - [ ] Are there new interface dependencies?
  - [x] There are no breaking changes in this PR.

If there are breaking changes, please **STOP** and consider the following:

- What other modules will these changes impact?
- Do JIRAs exist to update the impacted modules?
  - [ ] If not, please create them
  - [ ] Do they contain the appropriate level of detail?  Which endpoints/schemas changed, etc.
  - [ ] Do they have all they appropriate links to blocked/related issues?
- Are the JIRAs under active development?
  - [ ] If not, contact the project's PO and make sure they're aware of the urgency.
- Do PRs exist for these changes?
  - [ ] If so, have they been approved?

Ideally all of the PRs involved in breaking changes would be merged in the same day to avoid breaking the folio-testing environment.  Communication is paramount if that is to be achieved, especially as the number of intermodule and inter-team dependencies increase.

While it's helpful for reviewers to help identify potential problems, ensuring that it's safe to merge is ultimately the responsibility of the PR assignee.

### CODE REVIEW

File: src/main/java/org/folio/bulkops/util/Constants.java [hunk 1]
```diff
@@ -44,8 +44,8 @@ public class Constants {
   public static final String MSG_HOLDING_NO_CHANGE_REQUIRED_UNSUPPRESSED_ITEMS_UPDATED = "No change in value for holdings record required, associated unsuppressed item(s) have been updated.";
   public static final String MSG_HOLDING_NO_CHANGE_REQUIRED_SUPPRESSED_ITEMS_UPDATED = "No change in value for holdings record required, associated suppressed item(s) have been updated.";
   public static final String STAFF_ONLY = "(staff only)";
-  public static final String ADMINISTRATIVE_NOTES = "Administrative Notes";
-  public static final String ADMINISTRATIVE_NOTE = "Administrative Note";
+  public static final String CSV_COLUMN_NAME_ADMINISTRATIVE_NOTE = "Administrative note";
```
Comment (NikitaSedyx) [inline src/main/java/org/folio/bulkops/util/Constants.java:47]: are you sure you need 2 the same strings?

File: src/main/java/org/folio/bulkops/util/UnifiedTableHeaderBuilder.java [hunk 2]
```diff
@@ -83,7 +83,7 @@ private static Cell toUnifiedTableCell(Field field, Set<String> forcedVisible) {
     var column = field.getAnnotation(CsvCustomBindByName.class).column();
     return new Cell()
       .dataType(field.getAnnotation(UnifiedTableCell.class).dataType())
-      .value(ADMINISTRATIVE_NOTES.equalsIgnoreCase(column) ? ADMINISTRATIVE_NOTE : column)
+      .value(CSV_COLUMN_NAME_ADMINISTRATIVE_NOTE.equalsIgnoreCase(column) ? TABLE_HEADER_VALUE_ADMINISTRATIVE_NOTE : column)
```
Comment (NikitaSedyx) [inline src/main/java/org/folio/bulkops/util/UnifiedTableHeaderBuilder.java:86]: it looks a bit strange, if column = Administrative note -> set Administrative note, looks weird

Comment (DmytroBykov1) [reply src/main/java/org/folio/bulkops/util/UnifiedTableHeaderBuilder.java]: removed

File: src/main/java/org/folio/bulkops/util/Constants.java (hunk 1 above)
Comment (DmytroBykov1) [reply src/main/java/org/folio/bulkops/util/Constants.java]: removed

**Review by siarhei-charniak — APPROVED** [review]

File: src/main/java/org/folio/bulkops/util/UnifiedTableHeaderBuilder.java (hunk 2 above)
Comment (NikitaSedyx) [reply src/main/java/org/folio/bulkops/util/UnifiedTableHeaderBuilder.java]: are you sure it can be removed? what does worker generate?

Comment (DmytroBykov1) [reply src/main/java/org/folio/bulkops/util/UnifiedTableHeaderBuilder.java]: worker doesn't take a part in such a case
image

Comment (NikitaSedyx) [reply src/main/java/org/folio/bulkops/util/UnifiedTableHeaderBuilder.java]: worker generates file... check headers there and you'll understand why this function was implemented

Comment (DmytroBykov1) [reply src/main/java/org/folio/bulkops/util/UnifiedTableHeaderBuilder.java]: The snapshot above shows the exact place the table cell value is taken. the source of data is the file, produced by The worker.
The data mapping based on the position and entity type. Then the code analyzes the class field content by exact annotation.

for the HoldingRecord class field with position 18 the CsvCustomBindByName annotation column-attribute value is Administrative note

**Review by alekGbuz — APPROVED** [review]

File: src/main/java/org/folio/bulkops/util/UnifiedTableHeaderBuilder.java (hunk 2 above)
Comment (NikitaSedyx) [reply src/main/java/org/folio/bulkops/util/UnifiedTableHeaderBuilder.java]: again, worker generates another header, I'm 90% sure you'll introduce new issue? could you check all places in flow?

**Pull request merged**